README.md

```diff
@@ -1,5 +1,10 @@
 # Overview
-Project to build a base ubuntu image and deploy it to AWS
+Project to build a base ubuntu image for Sage Bionetworks science use and deploy it to AWS.
+The image is based on the latest Ubuntu Bionic release available at the point of build and also
+includes the following:
+- Python3 and supporting libs
```

Comment:
I'm betting that python 2.x has already been removed from all the AWS base images.  I would double check that and if they have then we should remove all the customization in this packer build to work with python 3.x

we should also remove python 2.7 stuff as well, https://github.com/Sage-Bionetworks-IT/packer-base-ubuntu-bionic/blob/master/src/playbook.yaml#L17

Reply:
I agree.... You're thinking just to inherit whatever the default Python install is on the Ubuntu Bionic release in the base AMI?

Reply:
Correction... looks like python2.7 is still the default on Bionic at the moment, so I think we need to keep the python3 logic.  Should we base it on a different release of Ubuntu?

Reply:
it probably depends on the base AWS AMI you are using.  I couldn't tell from the AMI ID that was updated.  if you are referencing bionic then probably keep as-is however if you are moving onto the latest release, , FocalFossa, then python 2 has been removed. https://wiki.ubuntu.com/FocalFossa/ReleaseNotes#Python3_by_default

Reply:
I updated the AMI-ID to the currently available Bionic AMI released by Canonical. Interesting about that Ubuntu doc... when I run a focal container and install python it definitely pulls python2:
`docker run -i -t ubuntu:focal /bin/bash` then `apt-get update && apt-get install python` results "Note, selecting 'python-is-python2' instead of 'python'"